### PR TITLE
TRIVIAL: Update translations

### PR DESCRIPTION
**Pull request by patrikbraborec**

### CODE REVIEW

File: src/translations/fr-FR.json [hunk 1]
```diff
@@ -166,6 +167,6 @@
     "mvf.operator.notBetween": "non compris entre",
     "mvf.operator.between.tooltip.bubble": "Les valeurs de début et de fin sont incluses.",
     "mvf.operator.notBetween.tooltip.bubble": "Les valeurs de début et de fin ne sont pas incluses.",
-    "mvf.treatNullValuesAsZeroLabel": "Traiter les valeurs vides en tant que 0",
+    "mvf.treatNullValuesAsZeroLabel": "Traiter les valeurs vides comme 0",
```
Comment (patrikbraborec) [inline src/translations/fr-FR.json:170]: Bugfix related to https://jira.intgdc.com/browse/BB-2235

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/7146/ : SUCCESS in 8m 56s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/3125/ : SUCCESS in 5s

File: src/translations/fr-FR.json (hunk 1 above)
Comment (xMort) [reply src/translations/fr-FR.json:170]: I tested the new string and it fixes the mentioned bug.

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/3735/ : UNSTABLE in 18m 19s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/7154/ : SUCCESS in 11m 44s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/1734/ : SUCCESS in 20s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/2550/ : SUCCESS in 19m 58s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/3742/ : SUCCESS in 15m 44s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/7162/ : SUCCESS in 6m 09s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/1741/ : SUCCESS in 17s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/2557/ : SUCCESS in 19m 21s

**Pull request merged**